### ISSUE TITLE
Bad code in plmm_checks

### ISSUE THREAD

**Issue by pbreheny** [pbreheny/plmmr#58]
Reproducible example:

``` r
library(plmmr)
dat <- list()
plmm(X = admix$X, y = admix$y)
#> Error in matrix(0, nrow = (p + length(non_genomic) + 1), ncol = ncol(std_scale_beta)):
#> invalid 'nrow' value (too large or NA)
```

The problem is here: https://github.com/pbreheny/plmmr/blob/5e663a57a24eeeed5bc4c5166bcc0c3f14c667c3/R/plmm_checks.R#L236-L240

Code like this should never appear in an R package. The code can't arbitrarily fail depending on whether the user has defined something in their global environment or not.

**Comment by pbreheny** [comment]
@tabpeter please fix this or close the issue if it is already fixed

**Comment by tabpeter** [comment]
this was fixed in version 4.0.0
